ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -96,8 +96,11 @@ protected override async Task<bool> ValidateAndSave()
             validationService.TrySetAndValidateDateOnModel(ClinicalDetails, nameof(ClinicalDetails.TreatmentStartDate), FormattedTreatmentDate);
             validationService.TrySetAndValidateDateOnModel(ClinicalDetails, nameof(ClinicalDetails.DeathDate), FormattedDeathDate);
             validationService.TrySetAndValidateDateOnModel(ClinicalDetails, nameof(ClinicalDetails.MDRTreatmentStartDate), FormattedMDRTreatmentDate);
-            validationService.ValidateYearComparisonOnModel(ClinicalDetails, nameof(ClinicalDetails.BCGVaccinationYear),
-                ClinicalDetails.BCGVaccinationYear, PatientBirthYear);
+            if (ClinicalDetails.BCGVaccinationYear != null)
```

Comment:
The behaviour for dynamic validation and serverside validation wasn't quite the same before - did some refactoring of this to fix this, also in validation service.

Reply:
Great, thanks! The closer they are together, the better I think!